app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) != 3:
+        return
+    mv_, source_path, destination_path = command.split()
```

Comment:
rename mv_ to command or cmd

Reply:
https://github.com/mate-academy/py-move-file/blob/master/checklist.md#code-efficiency
And provide this cheks